.github/workflows/labeler.yml

```diff
@@ -44,23 +44,26 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: "3.11"
+      - name: Set up UV
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
FYI Hynek has an action for this, handling caching and other stuff. And Astral also pulled somebody's action into their org recently too.

Reply:
(For context, @webknjaz is referring to https://github.com/hynek/setup-cached-uv)

Yeah, I was aware of his action (I follow him on Mastodon) and used it in the original incarnation of this PR.

I generally prefer to avoid pre-packaged GHAs unless they do something particularly complex that I wouldn't want to implement myself, and I don't think the caching is necessary for our use cases. See [this comment](https://github.com/hynek/setup-cached-uv/issues/7#issuecomment-2242290006) and the [`if-use-cache` option](https://github.com/hynek/setup-cached-uv?tab=readme-ov-file#if-use-cache). When we use locked dependencies, the caching doesn't save us any time and just adds more complexity.

So, since all I need to do is install the uv binary, I figured I'd do so using the upstream install script, which is also what setup-cached-uv uses.